server/endpoints/Profile.ts

```diff
@@ -77,6 +88,8 @@ export const getReviewsByStudentId: Endpoint<NetIdQuery> = {
     }
   },
 };
+<<<<<<< HEAD
```

Comment:
Not sure if you saw this, but I think you need to resolve the merge conflict here. I'm assuming we do want this function.

Reply:
Fixed!